src/common/lib/polkadot-provider/query/genetic-analyst-service.js

```diff
@@ -0,0 +1,10 @@
+export async function queryGetAllGeneticAnalystServices(api) {
+  let res = (await api.query.geneticAnalystServices.geneticAnalystServices.entries())
+  return res
+}
+
+export async function queryGetGeneticAnalystServiceById(api, accountId) {
+  let res = (await api.query.geneticAnalystServices.geneticAnalystServices(accountId))
+    .toHuman()
+  return res
```

Comment:
why are you using let instead of const and (? please update with better practice